lib/daylight/associations.rb

```diff
@@ -262,6 +263,8 @@ def define_cached_method method_name, options={}, &block
           association_hashcodes[method_name] = value.hash
 
           instance_variable_set ivar_name, value
+
+          return instance_variable_get(ivar_name)
```

Comment:
thanks to the above change, we can do this now. so whenever we used to do something like `@show.remote_start` you would get the same stale data. however now we return the freshest data so we can have more responsive remote methods so we can do something like

``` if @show.remote_start.state=='started'
        #some success logic
     end